pysparkling/sql/ast/tests/test_functions.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from pysparkling import Row
+from pysparkling.sql.ast.ast_to_python import parse_expression
+from pysparkling.sql.types import IntegerType, StructField, StructType
+
+ROW = Row(a=1, b=2, c=3)
+SCHEMA = StructType([
+    StructField("a", IntegerType()),
+    StructField("b", IntegerType()),
+])
+
+
+SCENARIOS = {
```

Comment:
Are all functions (currently) supported functions being covered?

Reply:
this is only for the datatypes. I didn't verify the functions yet.